tests/trainer/optimization/test_manual_optimization.py

```diff
@@ -1147,3 +1147,33 @@ def dis_closure():
 @RunIf(min_gpus=2, special=True)
 def test_step_with_optimizer_closure_with_different_frequencies_ddp_with_toggle_model(tmpdir):
     train_manual_optimization(tmpdir, "ddp", model_cls=TestManualOptimizationDDPModelToggleModel)
+
+
+def test_lr_scheduler_step_not_called(tmpdir):
+    """
+    Test `lr_scheduler.step()` is not called in manual optimization.
+    """
+    class TestModel(BoringModel):
+        def __init__(self):
+            super().__init__()
+            self.automatic_optimization = False
+
+    model = TestModel()
+    model.training_step_end = None
+    model.training_epoch_end = None
+
+    trainer = Trainer(
+        max_epochs=1,
+        default_root_dir=tmpdir,
+        limit_train_batches=1,
+        limit_test_batches=1,
+        limit_val_batches=1,
```

Comment:
should this be at least two?
fast_dev_run=2 maybe?
```suggestion
        fast_dev_run=2,
```

Reply:
Yes, it should be. Thanks.